CMakeLists.txt

```diff
@@ -111,10 +112,9 @@ endif ()
 
 
 # Set the default namespace
-if (OSL_NAMESPACE)
-    add_definitions ("-DOSL_NAMESPACE=${OSL_NAMESPACE}")
-endif ()
-message(STATUS "Setting Namespace to: ${OSL_NAMESPACE}")
+set (PROJ_NAMESPACE "${${PROJ_NAME}_NAMESPACE}")
```

Comment:
Hey @lgritz, I was referring to some of these changes in the main CMakeLists.txt file. i.e creating PROJ_NAMESPACE. This doesn't seem to end up in the Release branch if I set github to say point at RB 1.7 and the download a ZIP of this. I've now also tried cloning, checking out RB-1.7 and it's the same? I'm probably doing something dumb, but I can't see any of these new lines from this pull request?